### ISSUE TITLE
No foam commands working with v0.13.5

### ISSUE THREAD

**Issue by nossidge** [foambubble/foam#668]
- Foam version: 0.13.5
- Platform: Linux
- Issue occur on the [foam template](https://github.com/foambubble/foam-template) repo: Yes

**Summary**
When running Foam commands, they result in a "not found" error.

**Steps to reproduce**
1. Open any foam project 
2. Run "Foam: Show Graph" from the command palette
3. Or, type `Alt + d`

**Additional information**
Error text is:
```
Command 'Foam: Open Daily Note' resulted in an error (command 'foam-vscode.open-daily-note' not found)
```

Screenshot of the VS Code bottom-right errors:
![Screenshot from 2021-06-05 13-24-29](https://user-images.githubusercontent.com/3586214/120891745-1a651b00-c602-11eb-9811-f66adf2fed29.png)

Neither the keyboard shortcuts nor the Command Palette commands are working.

Seems to be the same symptoms that were described in the first comment of this issue: https://github.com/foambubble/foam/issues/363

It was working perfectly yesterday, but not today. Could this be due to the 0.13.5 update?

**Comment by riccardoferretti** [comment]
fixed in 0.13.7

**Comment by nossidge** [comment]
Wonderful, working perfectly again. Thanks for the quick turnaround!
